lib/commands/update.js

```diff
@@ -107,7 +112,7 @@ class UpdateCommand extends Command {
                 }
             }, {
                 title: 'Restarting Ghost',
-                skip: () => !argv.restart,
+                enabled: () => argv.restart === true,
```

Comment:
don't necessarily need the === true here, esp. since we have the boolean type in the option config

Reply:
Done!